src/plugins/management/public/plugin.tsx

```diff
@@ -167,6 +172,49 @@ export class ManagementPlugin
       });
     }
 
+    // Register the Settings app only if in serverless, until we integrate the SettingsApplication into the Advanced settings plugin
+    // Otherwise, it will be double registered from the Advanced settings plugin
+    if (_plugins.serverless) {
```

Comment:
nit: this variable was only named `_plugins` to avoid an `eslint` error about a variable not being used.
```suggestion
    if (plugins.serverless) {
```